test/runtests.jl

```diff
@@ -34,23 +34,23 @@ include("utils.jl")
 
 # Load synthetic data, models, generators
 synthetic = _load_synthetic()
-generators = generator_catalogue
+generators = Generators.generator_catalogue
 
 @testset "CounterfactualExplanations.jl" begin
     @testset "Data" begin
         include("data/data.jl")
     end
 
     @testset "Generators" begin
-        include("generators/counterfactuals.jl")
+        include("generators/generators.jl")
     end
 
     @testset "Probe" begin
         include("generators/probe.jl")
     end
 
     @testset "CLUE" begin
-        include("clue.jl")
+        include("generators/clue.jl")
     end
```

Comment:
CLUE and PROBE are already included in generators.jl and don't need to be included here.

Reply:
Done